mteb/tasks/Retrieval/fra/BSARDRetrieval.py

```diff
@@ -52,7 +52,7 @@ def load_data(self, **kwargs):
 
         self.queries = {
             self.metadata.eval_splits[0]: {
-                str(q["id"]): " ".join((q["question"] + q["extra_description"]))
+                str(q["id"]): q["question"] + q["extra_description"]
```

Comment:
What's the difference between these 2 implementations?

Reply:
exactly, we were ending up with queries that looked like this: "I  a m a  q u e r y"